### ISSUE TITLE
what is HTC++ mean?

### ISSUE THREAD

**Issue by fuweifu-vtoo** [microsoft/Swin-Transformer#113]
Dear author

I Want to know what is HTC++ mean? what is the difference between HTC and HTC++ mean?

do you have any explanation or url for me ?

**Comment by zeliu98** [comment]
Hi, you can find the explanation in the appendix (**A2.2. Object detection on COCO**) of the paper:

_For system-level comparison, we adopt an improved HTC [9] (denoted as HTC++) with instaboost [22], stronger multi-scale training [7] (resizing the input such that the shorter side is between 400 and 1400 while the longer side is at most 1600), 6x schedule (72 epochs with the learning rate decayed at epochs 63 and 69 by a factor of 0.1), soft-NMS [5], and an extra global self-attention layer appended at the output of last stage and ImageNet-22K pre-trained model as initialization._

**Comment by vadimkantorov** [comment]
@zeliu98 Am I right that HTC++ mean HTC together with improvements below:
* instaboost
* stronger multi-scale training
* 6x schedule
* soft-nms
* global self-attention layer

Or does it mean some other HTC++, and then improved with all these?

Is there an open-source mmdet/detectron2 implementation/config of HTC++? Thank you!

**Comment by vadimkantorov** [comment]
What is InstaBoost? The referred paper [[22] An image is worth 16X16 words: transformers for image recognition at scale ](https://arxiv.org/abs/2010.11929) contains no mention of InstaBoost technique
